src/Samples/SarifToCsv/Program.cs

```diff
@@ -150,6 +149,13 @@ public static void ConvertSarifLog(JsonSerializer serializer, string sarifFilePa
             writers["Location.Region.StartLine"] = (writer, result, pLoc) => { writer.Write(pLoc.Region?.StartLine ?? -1); };
             writers["Location.Region.Tags"] = (writer, result, pLoc) => { writer.Write(String.Join("; ", ((IEnumerable<string>)pLoc.Region?.Tags) ?? Array.Empty<string>())); };
 
+            writers["Properties.MatchState"] = (writer, result, pLoc) => 
+            {
+                string state;
+                if(!result.TryGetProperty("matchState", out state)) { state = null; }
```

Comment:
>matchState [](start = 43, length = 10)

since this isn't standard SARIF utilization, you should add a comment noting what one-off tool it originates with.

Reply:
I just removed it instead.